README.md

```diff
@@ -104,9 +104,45 @@ you want what behaviour. This is the recommended approach.
 
 The default timeout for contacting the server is 1 second, a request will be repeated once in case a timeout occurs, with circuit breakers applied.
 
-#### Create own authorization
+#### Create own authorization / using your own servers
 You can do so by subclassing OAuth2Action and `override def validateToken`.
 
+#### How to mock values in tests?
+Just create this in your project:
+
+```scala
+import scala.concurrent.duration._
+import scala.concurrent.Future
+import com.typesafe.config.Config
+
+import org.zalando.hutmann.authentication._
+trait Auth {
```

Comment:
Is there any reasons, why to not create this trait in the library? It seems, that every client will need it, because using OAuth2Action directly it's hard to test or remove OAuth.